src/app/story/story.component.html

```diff
@@ -4,7 +4,8 @@
 <div class="container-fluid">
   <div class="diagram" id="js-diagram">
     <ng-container *ngIf="story">
-      <app-storyrow [node]="node" [relatedNodes]="last ? story.meta.story_related_nodes_backward[i] : story.meta.story_related_nodes_forward[i]" [last]="last"
+      <app-storyrow [node]="node" [backwardSiblings]="story.meta.story_related_nodes_backward[i]"
+          [forwardSiblings]="story.meta.story_related_nodes_forward[i]" [last]="last"
```

Comment:
I'm not sure if we still need the `last` param since we have removed it from our API. Am I missing something?

Reply:
It's still used in `ngAfterViewInit` to call `connectStory` to draw all the lines.

Reply:
cool, makes sense